src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/UsersAndRolesMigrations/20220527101017_EES3388_RenameTestUserEmails_upsert.sql

```diff
@@ -0,0 +1,34 @@
+UPDATE AspNetUsers SET NormalizedUserName = 'EES-TEST.BAU1@education.gov.uk' WHERE NormalizedUserName = 'EES-BAU1@EDUCATION.GOV.UK';
+UPDATE AspNetUsers SET Email = 'EES-test.BAU1@education.gov.uk' WHERE Email = 'ees-bau1@education.gov.uk';
+UPDATE AspNetUsers SET NormalizedEmail = 'EES-TEST.BAU1@education.gov.uk' WHERE NormalizedEmail = 'EES-BAU1@education.gov.uk';
+UPDATE AspNetUsers SET UserName = 'ees-test.bau1@education.gov.uk' WHERE UserName = 'ees-bau1@education.gov.uk';
```

Comment:
Not saying you need to make this change as I think it's ok and not too important, but all of the updates in both files might have been easier to read if you updated all 4 fields changing in the `AspNetUsers` table in one update.

```
UPDATE AspNetUsers SET Email = 'EES-test.BAU1@education.gov.uk', NormalizedEmail = 'EES-TEST.BAU1@education.gov.uk', UserName = 'ees-test.bau1@education.gov.uk', NormalizedUserName = 'EES-TEST.BAU1@education.gov.uk'  WHERE UserName = 'ees-bau1@education.gov.uk';
```

20 separate updates would then become 5 updates.

Also I think it might be visually easier to follow the differences if you group the updates by the table that's changing, e.g. group all the `AspNetUsers` updates together, group all the `Users` updates together, etc, as each row will differ just by a few characters like BAU, Analyst, 1,2,3 etc.